patch-Disable-build-id.patch

```diff
@@ -0,0 +1,59 @@
+From 44b05af0e88ff9f1c7ccb1650b4318fc3df12554 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
+ <frederic.pierret@qubes-os.org>
+Date: Sun, 8 Nov 2020 10:42:31 +0100
+Subject: [PATCH] Disable build-id
```

Comment:
This is probably not acceptable upstream, as build-id is necessary for livepatching to work. But since we don't use livepatching, it's fine here.